### PR TITLE
Rename files and fix grammar

### PR DESCRIPTION
**Pull request by cdingm**
Closes Gamify-IT/issues#56

### CODE REVIEW

**Review by delvh — COMMENTED** [review]
Regarding the READMEs whose only purpose is to provide a TOC: I explicitly removed them because they don't work well with ReadTheDocs, each file would need to be manually disabled, and it is also annoying to keep them up to date.

File: dev-manuals/etc/glossary.md [hunk 1]
```diff
@@ -9,135 +9,210 @@ Imagine the phrase: `A xy is ...` when writing an explanation.
 ✅ Overworld: a world that ...
 ```
 
-[A](#a) [B](#b) [C](#c) [D](#d) [E](#e) [F](#f) [G](#g) [H](#h) [I](#i) [J](#j) [K](#k) [L](#l) [M](#m) [N](#n) [O](#o) [P](#p) [Q](#q) [R](#r) [S](#s) [T](#t) [U](#u) [V](#v) [W](#w) [X](#x) [Y](#y) [Z](#z)  
+[A](a) [B](b) [C](c) [D](d) [E](e) [F](f) [G](g) [H](h) [I](i) [J](j) [K](k) [L](l) [M](m) [N](n) [O](o) [P](p) [Q](q) [R](r) [S](s) [T](t) [U](u) [V](v) [W](w) [X](x) [Y](y) [Z](z)
+
```
Comment (cdingm) [inline dev-manuals/etc/glossary.md:13]: this change doesnt work, you cant click the Letters anymore

File: dev-manuals/etc/glossary.md [hunk 2]
```diff
@@ -9,135 +9,210 @@ Imagine the phrase: `A xy is ...` when writing an explanation.
...
+(a)=
```
Comment (cdingm) [inline dev-manuals/etc/glossary.md:14]: why this? i assume you wanted to associate the letters but it didnt work

**Review by cdingm — COMMENTED** [review]
The new linking doesnt work, what was the problem with the old?

File: dev-manuals/etc/glossary.md (hunk 1 above)
Comment (delvh) [reply dev-manuals/etc/glossary.md:13]: Not on GitHub, but on ReadTheDocs.

File: dev-manuals/etc/glossary.md (hunk 2 above)
Comment (delvh) [reply dev-manuals/etc/glossary.md:14]: That's ReadTheDocs only syntax to create an inner document reference.

File: dev-manuals/etc/glossary.md (hunk 1 above)
Comment (delvh) [reply dev-manuals/etc/glossary.md:13]: And I'm in favor of having working links on ReadTheDocs as they tend to be more user friendly.

Comment (delvh) [reply dev-manuals/etc/glossary.md:13]: So which of the two do we choose then?

**Review by delvh — APPROVED** [review]

**Comment by madebyTimo** [comment]
Is there still the need for a second review?

**Comment by cdingm** [comment]
I dont think so, i believe we said only one needed for docs? So I already merged it😅

**Pull request merged**